install_config/imagestreams_templates.adoc

```diff
@@ -104,12 +104,12 @@ ifdef::openshift-dedicated,openshift-enterprise[]
 |Type |Technology
 
 |Middleware Services
-a|- link:https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshift/3/single/red-hat-jboss-a-mq-for-openshift/[Red Hat JBoss A-MQ]
-- link:https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshift/3/single/red-hat-jboss-bpm-suite-intelligent-process-server-for-openshift/[Red Hat JBoss BPM Suite Intelligent Process Server]
-- link:https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshift/3/single/red-hat-jboss-brms-decision-server-for-openshift/[Red Hat JBoss BRMS Decision Server]
-- link:https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshift/3/single/red-hat-jboss-data-grid-for-openshift/[Red Hat JBoss Data Grid]
-- link:https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshift/3/single/red-hat-jboss-enterprise-application-platform-for-openshift/[Red Hat JBoss EAP]
-- link:https://access.redhat.com/documentation/en/red-hat-jboss-middleware-for-openshift/3/single/red-hat-jboss-fuse-integration-services-for-openshift/[Red Hat JBoss Fuse Integration Services]
+a|- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_a-mq/6.3/html-single/red_hat_jboss_a-mq_for_openshift/[Red Hat JBoss A-MQ]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_bpm_suite/6.4/html-single/red_hat_jboss_bpm_suite_intelligent_process_server_for_openshift/[Red Hat JBoss BPM Suite Intelligent Process Server]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_brms/6.4/html-single/red_hat_jboss_brms_realtime_decision_server_for_openshift/[Red Hat JBoss BRMS Decision Server]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_data_grid/7.1/html-single/data_grid_for_openshift/[Red Hat JBoss Data Grid]
+- link:https://access.redhat.com/documentation/en-us/red_hat_jboss_enterprise_application_platform/7.0/html-single/red_hat_jboss_enterprise_application_platform_for_openshift//[Red Hat JBoss EAP]
```

Comment:
I can see there are two "/" in the end of the link, it seems one of them needs to be removed, right?